main.js

```diff
@@ -1,3 +1,23 @@
+// This is used to pre-fill the grid, the dot is used to act as content to space it out.
+const grid = `
+    <div class="placeholder">.</div>
+    <div class="placeholder">.</div>
```

Comment:
i might be wrong but I think a space character would do the same as the dot here, and be a tidier solution